src/ts/container/component-container.ts

```diff
@@ -286,15 +267,79 @@ export class ComponentContainer extends EventEmitter {
         this.emit('tab', tab)
     }
 
+    /** @internal */
+    setVisibility(value: boolean): void {
+        if (this._boundComponent.virtual) {
+            if (this.virtualVisibilityChangeRequiredEvent !== undefined) {
+                this.virtualVisibilityChangeRequiredEvent(this, value);
+            }
```

Comment:
should there be a return here or is the code below supposed to run in the virtual component case too?

Reply:
There should **not** be a `return` here. After the component is made visible, it then needs to be positioned.  That is what the subsequent lines of code in this function do.